python/paddle/fluid/transpiler/details/__init__.py

```diff
@@ -17,3 +17,4 @@
 from .program_utils import *
 from .ufind import *
 from .checkport import *
+from .vars_distributed import *
```

Comment:
util functions in `io.py` also need same operations?

Reply:
just wait for a second, I am testing it locally.